### PR TITLE
 Finding last digit of nth term in squares of fibonacci number by C++ (#1696) 

### PR DESCRIPTION
**Pull request by RedEye00**
Issue #1696 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- adding new algorithm 

#### Languages Used:
-c++ 
#### Files Added:
- fib.cpp

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
@RedEye00  can you complete the checklist

**Comment by RedEye00** [comment]
sorry I forgot to complete but now please can u review.

File: fib.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include<iostream>
+int main(){
+    int a[60],n,i,c,b[60],j;
```
Comment (asha15) [inline fib.cpp]: please keep space after comma

File: fib.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include<iostream>
```
Comment (asha15) [inline fib.cpp]: keep a line space after this

File: fib.cpp [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include<iostream>
+int main(){
+    int a[60],n,i,c,b[60],j;
+    a[0]=0;
+    a[1]=1;
+    b[0]=0;
+    b[1]=1;
```
Comment (asha15) [inline fib.cpp]: keep space before and after =

File: fib.cpp [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+int main(){
+    int a[60],n,i,c,b[60],j;
+    a[0]=0;
+    a[1]=1;
+    b[0]=0;
+    b[1]=1;
+    for(i=2;i<60;i++){
+        a[i]=(a[i-1]+a[i-2])%10;
+        c=a[i];
+        b[i]=(c*c)%10;
```
Comment (asha15) [inline fib.cpp]: - keep space before and after operators 

- keep a space after ; in for loop 

- include some comments to explain the operation

File: fib.cpp [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    }
+    std::cout<<"Please enter the position Fibonacci number: ";
+    std::cin>>n;
+    std::cout<<"\nThe Last digit of square of this position : "<<b[n%60];
+    return 0;
```
Comment (asha15) [inline fib.cpp]: keep a line space before and after operator

File: fib.cpp [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+}
```
Comment (asha15) [inline fib.cpp]: add two sample input and  output

**Comment by asha15** [comment]
please include this file inside the appropriate folder, if the folder is not exist create a new one
please correct the file name
please include meaningful commit message

**Comment by RedEye00** [comment]
Sorry for the errors  ,I just wanted to ask whether I should close this PR request and debug my file and commit all changes and create new PR? or commit changes in existing file.

**Comment by asha15** [comment]
Do all the changes locally and push it into the same branch, no need to create a new pull request

**Comment by RedEye00** [comment]
plz review #2285 i have linked this issue to it sorry for your inconvenience and for late submission @asha15

**Comment by RedEye00** [comment]
plz review #2285 I have linked this issue to it sorry for your inconvenience and for late submission
@asha15

**Pull request closed without merging**